rllib/algorithms/appo/tf/appo_tf_learner.py

```diff
@@ -96,34 +95,34 @@ def compute_loss_per_module(
 
         behaviour_actions_logp_time_major = make_time_major(
             behaviour_actions_logp,
-            trajectory_len=self.rollout_frag_or_episode_len,
-            recurrent_seq_len=self.recurrent_seq_len,
-            drop_last=self.vtrace_drop_last_ts,
+            trajectory_len=self.hps.rollout_frag_or_episode_len,
```

Comment:
Using Hyperparameter object directly instead of creating duplicated information (before: copying all HP properties in the Learner c'tor into Learner properties).